starter-code/iteration-1/main.ts

```diff
@@ -1,11 +1,34 @@
 // Iteration 1
 // 1. Create an array of strings
 
+var tasksList: Array<string> = [];
+
 // 2. Create a function to add a task in to the array, call it `addTask`, the function has to accept a string and don't have to return anything.
+function addTask(nameTask:string){
+  tasksList.push(nameTask);
+  console.log("Task: " + nameTask + " inserted in the list");
+  return tasksList.length;
+}
 
 // 3. Create a function to list all tasks, it must show in the console de task.
+function listAllTasks(){
+  for(var i =0; i<tasksList.length; i++){
```

Comment:
you could improve this code by using a forEach and giving it a type of void:
`function listAllTasks(): void {
     this.tasks.forEach((task, index) => {
         console.log(task);
  });
}`